src/app/(auth)/(navigationsBarLayout)/problems/page.tsx

```
@@ -35,111 +46,116 @@ const ProblemsList = () => {
-              </div>
-            )}
-          </div>
-        )}
-      </section>
+              </FilterSelect>
+
+              <FilterSelect>
+                <label className="text-base">검색</label>
+                <div className="flex flex-row  w-full gap-5">
+                  <input
+                    placeholder="문제 제목 또는 번호 검색"
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)